<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTestForm</class>
 <widget class="QDialog" name="MyTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="3" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="2" column="4">
    <widget class="QtMaterialCheckBox" name="checkBox_2">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QtMaterialTextField" name="txt">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>TEST</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QtMaterialCheckBox" name="checkBox">
     <property name="text">
      <string>CheckBox</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QtMaterialRaisedButton" name="btn3">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QtMaterialRadioButton" name="radioButton">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtMaterialRadioButton" name="radioButton_2">
        <property name="text">
         <string>RadioButton</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QtMaterialRaisedButton" name="btn2">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QtMaterialFlatButton" name="btn1">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QtMaterialAutoComplete" name="txt2">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>TEST</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QLineEdit</extends>
   <header>QtMaterialTextField.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialRadioButton</class>
   <extends>QRadioButton</extends>
   <header>QtMaterialRadioButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialCheckBox</class>
   <extends>QCheckBox</extends>
   <header>QtMaterialCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header>QtMaterialFlatButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header>QtMaterialRaisedButton.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialAutoComplete</class>
   <extends>QLineEdit</extends>
   <header>QtMaterialAutoComplete.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
